src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.Apps/AppAction.cs

```diff
@@ -62,6 +62,6 @@ internal async Task Launch()
     public override CommandResult Invoke()
     {
         _ = Launch();
-        return CommandResult.GoHome();
+        return CommandResult.Dismiss();
```

Comment:
nit: should we clean up the file name -- 'AppAction' made sense in Wox land as Wox has actions, but since we are aiming for command palette vibes, probably should be `AppCommand.cs`

Reply:
We need to make a pass over a lot more than just this file - definitely better to do in a separate PR